opslevel/provider.go

```diff
@@ -51,25 +51,45 @@ func (p *OpslevelProvider) Schema(ctx context.Context, req provider.SchemaReques
 			},
 			"api_timeout": schema.Int64Attribute{
 				Optional:    true,
-				Description: "Value (in seconds) to use for the timeout of API calls made",
+				Description: "Value (in seconds) to use for the timeout of API calls made.  It can also be sourced from the OPSLEVEL_API_TIMEOUT environment variable.",
 				Sensitive:   false,
 			},
 		},
 	}
 }
 
+func (p *OpslevelProvider) ValidateConfig(ctx context.Context, req provider.ValidateConfigRequest, resp *provider.ValidateConfigResponse) {
+	var providerModel OpslevelProviderModel
+
+	resp.Diagnostics.Append(req.Config.Get(ctx, &providerModel)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	if providerModel.ApiUrl.IsNull() && os.Getenv("OPSLEVEL_API_TOKEN") == "" {
```

Comment:
why do we have to check for this here and in `configApiToken` ?

Reply:
Should this actually be the env var `OPSLEVEL_API_URL` ?  or is the usage of `providerModel.ApiUrl` wrong.

I'm so confused

Reply:
🤦‍♂️ typo from copy past